Modules/Microsoft365DSC/Examples/Resources/AADAdministrativeUnit/1-Create.ps1

```diff
@@ -23,7 +23,17 @@ Configuration Example
             MembershipRule                = "(user.country -eq `"Canada`")"
             MembershipRuleProcessingState = 'On'
             MembershipType                = 'Dynamic'
-            Ensure                        = 'Present'
```

Comment:
Why are we removing the Ensure parameter?

Reply:
Hey - I was hoping to catch issues like #4437 where a breaking bug was only uncovered because I left Ensure undefined.
Since #4475 this shouldn't be necessary on any resource anyway
It does seem to be fairly normal to be able to omit 'Ensure' in DSC resources. If you'd prefer consistency, I could remove Ensure from all other 'create' / 'update' examples - or add it back here if you prefer. (Of course this would prevent other bugs like the one in #4437 being unearthed.

Reply:
Ok thanks for the clarifications. Let's leave it as is for now then.